src/init.sh

```diff
@@ -76,9 +76,9 @@ service rsyslog start
 
 if [ -z $test_env ]
 then
-	/usr/local/bin/istgt -R 20
+	IO_MAX_WAIT_TIME=120 /usr/local/bin/istgt -R 20
```

Comment:
Why are we setting `IO_MAX_WAIT_TIME=120` in both cases?

Reply:
both the places we are setting the env.. requirement is to set the env, not the value